### ISSUE TITLE
Suggestion for Quick screenshot

### ISSUE THREAD

**Issue by manuelmagan** [TenderOwl/Frog#81]
Hi! I'm impressed by the app. Thank you!

I would suggest a second "Take a screenshot" option, something like a "Quick screenshot". It wouldn't ask for type of screenshot we want (it may be configured previously), neither to review it prior sharing to Frog. 

Skipping those two steps would come handy in repetitive tasks. Like those situations where we have to scan different parts of the same text; we always use the same type of screenshot and we know Frog will perform well.

Thank you again!

**Comment by amka** [comment]
@gregorni you're right but there's a catch: @mmgn897 it is possible to use the screenshot tool via dbus and it doesn't need to request that permission. While this is a better solution in the case of UX, it's not that good If we talk about security.

**Comment by DaPigGuy** [comment]
As of xdg-desktop-portal 1.15.0, non-interactive screenshots are now possible for skipping the portal dialogs. From my testing (changing the screenshot flag to `NONE` [here](https://github.com/TenderOwl/Frog/blob/master/frog/screenshot_backend.py#L78)), the window has to be visible when the screenshot is requested. Frog would still need UI for selecting an area of the screen however

**Comment by amka** [comment]
I did, actually. A lot of googling took me to the thread about this. So, the central part, for now, is implementing the screenshot UI. It will take some time :)
